components/test_sst_importer/src/lib.rs

```diff
@@ -5,19 +5,36 @@ use std::path::Path;
 
 use crc::crc32::{self, Hasher32};
 use engine_rocks::RocksEngine;
+use engine_rocks::RocksSstReader;
+use engine_rocks::RocksSstWriter;
+use engine_rocks::RocksSstWriterBuilder;
 use engine_traits::KvEngine;
+use engine_traits::SstReader;
+use engine_traits::SstWriter;
+use engine_traits::SstWriterBuilder;
 use kvproto::import_sstpb::*;
 use uuid::Uuid;
 
 use engine::rocks::util::new_engine;
-use engine::rocks::SstWriterBuilder;
 use std::sync::Arc;
 
+pub use engine_rocks::RocksEngine as TestEngine;
+
 pub fn new_test_engine(path: &str, cfs: &[&str]) -> RocksEngine {
     let db = new_engine(path, None, cfs, None).expect("rocks test engine");
     RocksEngine::from_db(Arc::new(db))
 }
 
+pub fn new_sst_reader(path: &str) -> RocksSstReader {
+    RocksSstReader::open(path).expect("test sst reader")
```

Comment:
I think these two functions should return results instead of panicking (it's a hidden panic, and it seems easy to trigger, just pass a bad path).

Reply:
This is test utility function so it's good (we could use RUST_BACKTRACE=1 to find who triggered the error).

Reply:
Yes, test code.

Reply:
Yes, this is test code.

Reply:
Oh right, of course (missed the crate name).